src/components/SwitchUserRoleDialog.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <v-dialog max-width="504" persistent>
+        <v-card>
+            <v-card-item>
+                <v-card-title>Switch User Role</v-card-title>
+                <v-card-subtitle
+                    >TODO Explain the purpose of this dialog to the
```

Comment:
TODO?

Reply:
Well... good thing we do these code reviews... xD